src/commands/index.ts

```diff
@@ -228,6 +229,8 @@ const commands: CommandDefinition[] = [
     salty,
```

Comment:
There is an extra `salty,` out of line.

Would suggest checking you are in line and moving your imports to the end of the list